### PR TITLE
Final merge

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

**Comment by SteveLasker** [comment]
Not directly covered in this PR update, but a comment on the overall use case:

- `Micro Coding Wizards (MCW)` - I happen to know MCW represents a few other companies (https://www.mcw.com/, https://www.mcwtech.com/). I might suggest we use some of the sample companies that we already have domains for, or can acquire to avoid conflicts with existing companies. 
- The VDR example, at first, seemed redundant with the SBOM scenario. I'd suggest the remaining difference is a VEX is a continual incremental disclosure of information, over the life of the artifact.

I've created PR #5 for review

File: draft-birkholz-scitt-software-supply-chain-use-cases.md [hunk 1]
```diff
@@ -50,23 +53,41 @@ Generalized Software Supply Chain Use Case Descriptions
...
+oducts and analytics solutions will assist in managing compliance and assessing risk to help prevent and detect supply chain attacks across the entire software lifecycle while prioritizing data privacy.
 
 ## Terminology {#terms}
 
 {::boilerplate bcp14-tagged}
 
 # Generic Problem Statement
 
-Supply chain security is paramount to protecting critical infrastructure, aerospace, and defense and avoiding impacts on security, the economy, public health, and safety. It has historically focused on risk management practices to safeguard logistics, meet compliance regulations, demand forecasts, and optimize inventory. While these elements are foundational to a healthy supply chain, an integrated cyber security-based perspective of the software supply chains remains broadly undefined. Recently, the global community has experienced numerous supply chain attacks by cybercriminals targeting weaknesses in software supply chains. As illustrated in {{lifecycle-threats}}, a software supply chain attack may leverage one or more lifecycle stages and directly or indirectly target the component.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-software-supply-chain-use-cases.md:61]: is this text intentionally removed?

File: draft-birkholz-scitt-software-supply-chain-use-cases.md [hunk 2]
```diff
@@ -50,23 +53,41 @@ Generalized Software Supply Chain Use Case Descriptions
...
+As illustrated in {{lifecycle-threats}}, a supply chain attack may leverage one or more lifecycle stages and directly or indirectly target the component.
 
-<!-- ![SCITT_SW_Use_Case](https://user-images.githubusercontent.com/100775190/197196387-f8f835ba-7023-4223-98c9-3d8ec197e658.svg) -->
 ~~~~
-generic supply chain threats diagram here
+threat chain diagram here
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-software-supply-chain-use-cases.md:76]: is this intentionally changed back?

File: draft-birkholz-scitt-software-supply-chain-use-cases.md [hunk 3]
```diff
@@ -78,6 +99,7 @@ deployment chain diagram here
 {: #deployment-chain title="Deployment Example of SCITT in Software Development"}
 
 # Software Supply Chain Use Cases
+SCITT fundamental architecture emphasizes heavily on introducing transparency and accountability to the entire steps involved in software delivery. With the advancement of Software Bill of Materials (SBOM) as a standard practice for managing software deliveries the below use case will take SBOM as an example to illustrate how a SCITT based system can augment security built using SBOMs.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-software-supply-chain-use-cases.md:103]: This reads as if only one SBOM use case follows, but there are three software supply chain use cases?

File: draft-birkholz-scitt-software-supply-chain-use-cases.md [hunk 4]
```diff
@@ -96,18 +118,30 @@ The following steps are performed by the Software Producer:
 1. Create the “final SBOM” listing all the components contained in the final distribution package of a software product, which the customer will install into their environment.
 The SBOM must follow NTIA minimum elements for SBOM’s and other NTIA and NIST recommendations for SBOM, to meet Executive Order 14028 and OMB M-22-18 requirements.
 (Co)SWID, SPDX or CycloneDX SBOM formats are acceptable for this artifact. 
-2. Digitally sign the SBOM artifact.
-3. Place the SBOM and digital signature artifacts within an access-controlled location, i.e. a customer portal, and provide the end consumer with a link to these artifacts for downloading to the customers environment.
+2. The encoding of SBOM is specific to the protocol in question.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-software-supply-chain-use-cases.md:122]: Why suddenly protocol?

**Comment by rjb4standards** [comment]
Steve, VEX is not supported in NIST Guidance. You should consider using VDR, Vulnerability Disclosure Report

File: draft-birkholz-scitt-software-supply-chain-use-cases.md [hunk 5]
```diff
@@ -96,18 +118,30 @@ The following steps are performed by the Software Producer:
...
+3. Makes a claim to establish its own identity as a producer of the overall SBOM.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-software-supply-chain-use-cases.md:123]: 'Makes'? A bit colloquial and if... why not 'make'?

File: draft-birkholz-scitt-software-supply-chain-use-cases.md [hunk 6]
```diff
@@ -96,18 +118,30 @@ The following steps are performed by the Software Producer:
...
+4. The issuer in certain situations can make further claims (example claims that relate to steps that might assist independent building of the software components, to arrive at the final deliverable).
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-software-supply-chain-use-cases.md]: This leaves a lot to the imagination. Why would all these claims clumped together and not be... stand-alone claims?

File: draft-birkholz-scitt-software-supply-chain-use-cases.md [hunk 7]
```diff
@@ -96,18 +118,30 @@ The following steps are performed by the Software Producer:
...
+5. Once all the claim construction is complete, make a CBOR data payload of all the claims.
+6. Signs the payload using SCITT recommended signing scheme. For now this will be COSE signing.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-software-supply-chain-use-cases.md]: Sign?

File: draft-birkholz-scitt-software-supply-chain-use-cases.md [hunk 8]
```diff
@@ -96,18 +118,30 @@ The following steps are performed by the Software Producer:
...
+7. Submits the COSE signed object to the transparent registry.
+8. Receives the return receipts (A COSE Countersignature object, containing the inclusion proofs, the original COSE payload and a signature from the transparency service).
+9. Producer would then ship the software along with the return receipts to the Software Consumer.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-software-supply-chain-use-cases.md]: The test style seems a little bit out of place. Also, why is COSE (a solution) mentioned in a use case?

File: draft-birkholz-scitt-software-supply-chain-use-cases.md [hunk 9]
```diff
@@ -96,18 +118,30 @@ The following steps are performed by the Software Producer:
...
 
 ### Consumer Actions
 
-A software consumer, in this case DOI, obtains a digitally signed SBOM artifact from a software vendor, which initiates the following risk assessment process:
+A software consumer, in this case DOI, receives the software deliverable along with return receipts from the Producer.
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-software-supply-chain-use-cases.md]: Why is the solution concept of a receipt introduced here in the use case scope?

File: draft-birkholz-scitt-software-supply-chain-use-cases.md [hunk 10]
```diff
@@ -96,18 +118,30 @@ The following steps are performed by the Software Producer:
...
+2. It will decode the return receipts to get the inclusion proof.
+3. Using inclusion proofs it will cryptographically evaluate the presence of Supply Chain Artifacts (including SBOM)  for the software deliverable in the Transparency Service.
+4. It will decode the CBOR Payload to get the SBOM and other claims made by the issuer.
+5. It may do additional verification to check that the identity claim value matches the identity of the issuer from where it received the software or the trust chain linking to the issuer.
+6. A format aware consumer can decode the SBOM contents and do the SBOM specific verification steps to further augment the trustworthiness of the received deliverable.
+7. The consumer can decode other claims to establish augmented trust on the received software. For example decode the specific claim to get the required instructions to build the software it has received. This way it can compare the computed hash for the binary compiled locally against the received hash from SCITT registry. Any mis-match can be reported as an inconsistency.
+
+#### SBOM specific actions
```
Comment (henkbirkholz) [inline draft-birkholz-scitt-software-supply-chain-use-cases.md:143]: Is it necessary to introduce another section depth here?

**Comment by SteveLasker** [comment]
> Steve, VEX is not supported in NIST Guidance. You should consider using VDR, Vulnerability Disclosure Report

Oh, sorry, I transformed. I'm updating my comment from VEX to VDR.

**Review by henkbirkholz — COMMENTED** [review]
Not sure, if we can merge that yet. Sorry for the TZ distortion due to traveling.

File: draft-birkholz-scitt-software-supply-chain-use-cases.md (hunk 1 above)
Comment (SteveLasker) [reply draft-birkholz-scitt-software-supply-chain-use-cases.md:61]: @yogeshbdeshpande, was this intentional, for brevity's sake?

Comment (yogeshbdeshpande) [reply draft-birkholz-scitt-software-supply-chain-use-cases.md:61]: @SteveLasker @henkbirkholz I was just merging the same in the PR coming from the main line: Let me revisit the same!

**Comment by yogeshbdeshpande** [comment]
Closing this PR as all the changes have been taken in the recent edits made by Henk

**Pull request closed without merging**